prediction_market_agent/db/db_storage.py

```diff
@@ -24,27 +28,50 @@ def _initialize_db(self) -> None:
         """
 
         # trick for making models import mandatory - models must be imported for metadata.create_all to work
-        logger.debug(f"tables being added {LongTermMemories}")
+        logger.debug(f"tables being added {LongTermMemories} {Prompt}")
         SQLModel.metadata.create_all(self.engine)
 
-    def save_multiple(
+    def save_multiple(self, items: list[Any]) -> None:
+        with Session(self.engine) as session:
+            session.add_all(items)
+            session.commit()
+
+    def save_multiple_long_term_memories(
         self,
         task_description: str,
         history: list[Dict[str, Any]],
     ) -> None:
         """Saves data to the LTM table with error handling."""
 
+        history_items = [
+            LongTermMemories(
+                task_description=task_description,
+                metadata_=json.dumps(history_item),
+                datetime_=utcnow(),
+            )
+            for history_item in history
+        ]
+
+        self.save_multiple(history_items)
+
+    def load_latest_prompt(
+        self, session_identifier: str | None = None
+    ) -> Prompt | None:
+        """
+        Queries the prompts table by session identifier with error handling
+        """
+        if not session_identifier:
+            session_identifier = PROMPT_DEFAULT_SESSION_IDENTIFIER
```

Comment:
Why not just make it a default argument in the function definition? But should there be a default value?